src/en/getting-started.md

```diff
@@ -49,52 +57,40 @@ particular cloud provider. This is done by generating and editing the following
 ## Windows
     %LOCALAPPDATA%/Juju
```

Comment:
June 12 @ EDT on #juju
14:33 <    hazmat> pmatulis: on windows it goes to the user's HOMEDRIVE + HOMEPATH env var then .juju
14:42 <  pmatulis> hazmat: so the online docs are wrong then. they currently show
14:42 <  pmatulis> %LOCALAPPDATA%/Juju
*silence*